src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -21,4 +22,12 @@
 
     public Optional<List<Book>> findByPublisherAndGenreAndYear(String publisher, String genre, String year);
 
+    @Query("SELECT b from Book b "
```

Comment:
This is good, can you add it documentation?, please

Reply:
corrected